### ISSUE TITLE
Random Seed requirement

### ISSUE THREAD

**Issue by ChrisPalmerNZ** [datquocnguyen/jLDADMM#1]
I've found the DMM model very helpful for identifying topics in twitter feeds - thank-you very much for creating it! I'm now experimenting with the use of the DMM model in a workflow that conducts inference on new data I have found that each run re-interprets my data and the arrangement of the topics differently. I have used a pre-trained 14 topic model and specified 14 topics for my output - ideally this should use the same clustering as my original model, or at least a consistent output between each run over the same data, but it differs each time. Can you add a random seed parameter that allows us to override the random initialization of topic assignments? By comparison, the MALLET topic modelling system allows for this and I can get consistent results when I pass in random seed...

**Comment by datquocnguyen** [comment]
Thanks. I've added the random seed option "-seed". See [this commit](https://github.com/datquocnguyen/jLDADMM/commit/3aa39ac74da8077afbccd75a6417a3efcb257956) for more details.

**Comment by ChrisPalmerNZ** [comment]
Thank-you _very_ much Dat - I am so impressed at how quickly you implemented this! I can't wait to get back to my project tonight and test it out! Thanks again... Chris

**Comment by ChrisPalmerNZ** [comment]
Thanks again for this - may I suggest you also do this for the LFTM project?

Kind Regards
Chris

**Comment by datquocnguyen** [comment]
I might do it later. For jLDADMM, I added only 4 code lines (see [this commit](https://github.com/datquocnguyen/jLDADMM/commit/3aa39ac74da8077afbccd75a6417a3efcb257956)). You can do exactly the same with LFTM and then recompile with `ant`.

**Comment by ChrisPalmerNZ** [comment]
Thanks Dat - awesome!

**Comment by ChrisPalmerNZ** [comment]
Hi Dat, I am struggling to use this on my Windows 10 machine. Even after upgrading to the latest java offering it appears out of sync with the version you've used to compile the JAR file - I get this message: "Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.UnsupportedClassVersionError: jLDADMM has been compiled by a more recent version of the Java Runtime (class file version 53.0), this version of the Java Runtime only recognizes class file versions up to 52.0". I believe I have java 8 and maybe you have java 9? If I type java -version I get:  java version "1.8.0_212" Is it possible to re-compile in a version compatible with my system (I believe I'm probably like many windows users)?

**Comment by datquocnguyen** [comment]
From the readme.md file: "The users can recompile the source codes by simply running `ant` (it is also expected that `ant` is already installed)" Hope it helps. Btw, I also updated `.jar` file few hours ago, which was compiled with java version 1.8.0_152.

**Comment by ChrisPalmerNZ** [comment]
Thanks for that, the new version is compatible with my system. In the meantime I had installed IntelliJ IDEA Community Edition, figured out through trial and error what is the latest Java version (1.8.0_202) I can successfully install with a JDK into my PC, and also figured out how to build the JAR file - so yes in a roundabout way I followed the instructions to rebuild with `ant`. I am completely new to java, so I think I've gone the long way around, there should be a simple process to achieve this with just `ant` - next thing to learn!
